### PR TITLE
Fix context leaks

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

File: .golangci.yml
Comment (AaronH88) [inline .golangci.yml:1]: Why add all this?

**Comment by AaronH88** [comment]
I think the name of the PR can do without the word "possible" :smile:

File: .golangci.yml
Comment (thom-at-redhat) [reply .golangci.yml:1]: There was a partial commented out list of linters, including deprecated ones.  I brought it up to date with all current linters (as I had mentioned this morning).  If you would prefer the commented lines to be removed, I'm fine with that - just let me know..

**Comment by thom-at-redhat** [comment]
> I think the name of the PR can do without the word "possible" 😄

Done

File: .golangci.yml
Comment (thom-at-redhat) [reply .golangci.yml:1]: @aaronh88 ^

Comment (AaronH88) [reply .golangci.yml:1]: Yes please, remove them.

Im personally not a fan of adding code then commenting it out :smile:

Comment (thom-at-redhat) [reply .golangci.yml:1]: Done,  we need to review the excludes in this file at a later point also

**Review by AaronH88 — APPROVED** [review]
lgtm

**Pull request merged**